administration/src/bp-modules/applications/VerificationsQuickIndicator.tsx

```diff
@@ -28,7 +28,7 @@ const VerificationQuickIndicator = memo(({ verifications }: { verifications: App
           {t('verified/pending/rejected')}
         </div>
       }>
-      <UnFocusedDiv>
+      <UnFocusedDiv data-testid='indicator-container'>
```

Comment:
I think this is not used anymore 
```suggestion
      <UnFocusedDiv>
```